### PR TITLE
Feature/faq 0.9.0 20170607

### PR DESCRIPTION
**Pull request by soyerr**
Draft for FAQ API
Every edit/remove actions should be handled on client side so request won'be send. Service should be able to save all tuples in db and get all tuples from db

### CODE REVIEW

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/AcpFAQService.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.frontend.api.faq;
+
+import org.jbb.frontend.api.faq.model.AcpFAQTuple;
+
+import java.util.List;
+
+public interface AcpFAQService {
```
Comment (baart92) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/AcpFAQService.java]: Why all interfaces has "Acp" prefix? It should be removed, it is not connected with acp directly. And then use camel case, so not `FAQService` but `FaqService` name is valid

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQCategory.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  You may obtain a copy of the License at
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.jbb.frontend.api.faq.model;
+
+/**
+ * Created by Piotr on 2017-06-07.
+ */
```
Comment (baart92) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQCategory.java]: Remove those comments

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQQuestion.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ */
+
+package org.jbb.frontend.api.faq.model;
+
+/**
+ * Created by Piotr on 2017-06-07.
+ */
+public interface AcpFAQQuestion {
+
+    String getQuestion();
```
Comment (baart92) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQQuestion.java]: Maybe just `getContent()`? Now you can have code like: `question.getQuestion()` - it is not clear for me

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQTuple.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ */
+
+package org.jbb.frontend.api.faq.model;
+
+/**
+ * Created by Piotr on 2017-06-07.
+ */
+public interface AcpFAQTuple {
+
+    AcpFAQCategory getFAQCategory();
```
Comment (baart92) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQTuple.java]: FaqCategory should be owner of FaqTuples. I mean, FaqCategory should have list of tuples for me

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/AcpFAQService.java (hunk 1 above)
Comment (soyerr) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/AcpFAQService.java]: They are connected.  
FAQ as a <K,V> pair is display to e2e user however FAQ settings are located in ACP bookmark. 
Only Administrator can define new or delete FAQ question ... so ye, it's connected

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQCategory.java (hunk 2 above)
Comment (soyerr) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQCategory.java]: Fix

File: domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQQuestion.java (hunk 3 above)
Comment (soyerr) [inline domain-api/jbb-frontend-api/src/main/java/org/jbb/frontend/api/faq/model/AcpFAQQuestion.java]: Ok

**Pull request merged**